src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import Help, { HelpProps } from "../../../src/components/help/help.component";
+import HelpComponentTest from "./component.test-pw";
+import Box from "../../../src/components/box";
+import {
+  getDataElementByValue,
+  icon,
+} from "../../../playwright/components/index";
+import helpComponent from "../../../playwright/components/help";
+import {
+  checkAccessibility,
+  containsClass,
+} from "../../../playwright/support/helper";
+import { COLOR, CHARACTERS } from "../../../playwright/support/constants";
+import { HooksConfig } from "../../../playwright";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const tooltipText = "Some helpful text goes here";
+const colors = [
+  ["orange", COLOR.ORANGE],
+  ["red", COLOR.RED],
+  ["black", COLOR.BLACK],
+  ["brown", COLOR.BROWN],
+];
+
+test.describe("when focused", () => {
+  test("should have the expected styling when the focusRedesignOptOut is false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<HelpComponentTest />);
+
+    const elementLocator = await helpComponent(page);
+    await elementLocator.focus();
+
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+
+  test("should have the expected styling when the focusRedesignOptOut is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<HelpComponentTest />, {
+      hooksConfig: { focusRedesignOptOut: true },
+    });
+    const elementLocator = await helpComponent(page);
+    await elementLocator.focus();
+
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "outline",
+      "rgb(255, 188, 25) solid 2px"
+    );
+  });
+});
+
+test.describe("Testing Help component properties", () => {
+  testData.forEach((ariaLabel) => {
+    test(`should check ariaLabel as ${ariaLabel}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest ariaLabel={ariaLabel} />);
+      await expect(helpComponent(page)).toHaveAttribute(
```

Comment:
**suggestion: (non-blocking)** it would be better to add a space to separate the mount from the assertion to follow the same pattern as with the other test cases.